lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1372,13 +1372,15 @@ paths:
         - EnodeBs
       parameters:
         - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - $ref: './orc8r-swagger-common.yml#/parameters/page_size'
```

Comment:
While we're here, go ahead and refactor all the existing swagger instances/definitions of pagesize/pagetoken to use the common.yml definitions